docker-entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
+
+if [[ ! -d "$DATA_DIR" ]]; then
+        exit 1
+fi
+
+./manage.py initialize
+
+chown -R $UID:$GID $DATA_DIR
```

Comment:
For this service, since it only has one file on it, it's OK to always run a `chown -R`. For other repositories that may have a lot of data under the volume, you can't use `-R` every container boot because it would take a lot of time